sdk/download.go

```diff
@@ -271,6 +272,11 @@ func UpdateFile(file, url string, client *http.Client) error {
 	tempFile := t.Name()
 	defer os.Remove(tempFile)
 
+	// base the file's mode on umask instead of just 0600
+	umask := syscall.Umask(0777)
```

Comment:
Perhaps this step should be done in `init()` so we aren't changing process-wide state when threads may be involved.